### DISCUSSION TITLE
Error getting package info

### DISCUSSION THREAD

**Discussion by kesavan22** [capawesome-team/capacitor-firebase#638] [Q&A]
Recently install the firebase authentication in my project, following the instructions mentioned in the documentation, but facing while signing in with google attached the Screenshot Below.

Any help would be greatly appreciated.

![image](https://github.com/capawesome-team/capacitor-firebase/assets/91103199/f4063b7b-76e6-4a48-9578-065ea772e99d)

**Comment by robingenz** [comment]
This is a duplicate of:

- https://github.com/capawesome-team/capacitor-firebase/issues/220
- https://github.com/robingenz/capacitor-firebase-authentication/issues/97
- https://github.com/robingenz/capacitor-firebase-authentication-demo/issues/164
- https://github.com/capawesome-team/capacitor-firebase/discussions/415
- https://github.com/capawesome-team/capacitor-firebase/discussions/379
- https://github.com/robingenz/capacitor-firebase-authentication/issues/97

Please search for existing issues/discussions before creating a new one.
